server/FuzzServer/sfreerdp.c

```diff
@@ -0,0 +1,937 @@
+/**
```

Comment:
Fuzzing velocity is quite low - 3 runs per second:
```
#79     NEW    cov: 2222 ft: 4453 corp: 25/55b lim: 4 exec/s: 3 rss: 55Mb L: 3/4 MS: 1 ChangeByte-                                                                    
```

> A typical good fuzz target will have an order of 1000 executions per second per one CPU core on average (exec/s) or more. For lightweight targets, 10000 exec/s or more.

> If your fuzz target has less than 10 exec/s you are probably doing something wrong. We recommend to profile fuzz targets and eliminate any obvious hot spots.



https://github.com/google/fuzzing/blob/master/docs/good-fuzz-target.md#speed